charts/rancher-eks-operator/1.0.101/templates/deployment.yaml

```diff
@@ -18,3 +18,10 @@ spec:
       - name: eks-operator
         image: {{ template "system_default_registry" . }}{{ .Values.eksOperator.image.repository }}:{{ .Values.eksOperator.image.tag }}
         imagePullPolicy: IfNotPresent
+        env:
+        - name: HTTP_PROXY
+          value: {{ .Values.httpProxy }}
```

Comment:
Nit: Seems that it is ok if these are set to blank, but you could wrap in conditional so they aren't set at all if blank.

Reply:
The rancher code marshal the struct into yaml and passes it to helm without checking if each variable is empty. Therefore, they will never be empty as rancher uses it. I can do the check though.

Reply:
Na its fine